plugins/archlinux/README.md

```diff
@@ -128,6 +136,8 @@ upgrades were available. Use `pacman -Que` instead.
 | Alias   | Command                        | Description                                                       |
 |---------|--------------------------------|-------------------------------------------------------------------|
 | yaconf  | `yay -Pg`                      | Print current configuration                                       |
+| yaclean | `yay -Sc`                      | Clean out old and unused caches and packages                      |
+| yaclr   | `yay -Scc`                     | Remove all files from the cache                                   |
```

Comment:
**Couldn't check what `-Scc` does**. Hard to link to docs, [here's a rendered man page](https://linuxcommandlibrary.com/man/yay#extended-pacman-operations) and [the source](https://github.com/Jguer/yay/blob/02d33f63c044906e014bfef7c96ed3e0c2586d76/doc/yay.8#L47-L50):
> ## EXTENDED PACMAN OPERATIONS
> -S, -Si, -Sl, -Ss, -Su, -Sc, -Qu
> > These operations are extended to support both AUR and repo packages.
> 
> -Sc
> > Yay will also clean cached AUR package and any untracked Files in the cache. Cleaning untracked files will wipe any downloaded sources or built packages but will keep already downloaded vcs sources.

Reply:
The documentation for the -Scc flag is given in the [arch wiki](https://wiki.archlinux.org/title/pacman)

> To remove all files from the cache, use the clean switch twice, this is the most aggressive approach and will leave nothing in the cache folder:

```
pacman -Scc
```

Reply:
But that's for pacman. Does `yay` support the same exact pacman flags?

Reply:
yeah, I tried the same for all the other helpers including yay. If you scroll all the way to the start of this conversation @ratijas asked me specifically to try the same for all the helpers and add the same flag for all of them. Tried and tested on Yay, Trizen , Pacaur and Aura.